app/controllers/admin_public_body_controller.rb

```diff
@@ -142,10 +142,12 @@ def mass_tag_add
   def missing_scheme
     # There might be a way to do this in ActiveRecord, but I can't find it
     @public_bodies = PublicBody.find_by_sql("
-            SELECT a.id, a.name, a.url_name, COUNT(*) AS howmany
```

Comment:
> Is my understanding of this correct?

I think so...
* 2 of the 3 queries in the [`missing_scheme` action](https://github.com/mysociety/alaveteli/blob/0.29.0.0/app/controllers/admin_public_body_controller.rb#L141-L155) break when we drop the translated columns from the `public_bodies` table (easily verifiable by copy-pasting the code into the console, and I've only just picked up on the second failure)
* there are no tests for this action
* it's an admin action - if we don't link to it, maybe it's not being used